adapter/src/dummy.rs

```diff
@@ -1,7 +1,7 @@
 #![deny(clippy::all)]
 #![deny(rust_2018_idioms)]
```

Comment:
why is this needed?

Reply:
https://github.com/rust-lang/rust/issues/52047